docs/source/tutorials/hospital_admissions_model.qmd

```diff
@@ -213,12 +213,12 @@ rtproc = MyRt()
 
 # we place a log-Normal prior on the concentration
 # parameter of the negative binomial.
-nb_conc_rv = metaclass.DistributionalRV(
-    dist.LogNormal(
-        loc=jnp.log(1),
-        scale=jnp.log(10),
-    ),
+nb_conc_rv = metaclass.TransformedRandomVariable(
     "concentration",
+    metaclass.DistributionalRV(
+        dist.TruncatedNormal(low=0.01), "concentration_raw"
```

Comment:
Minor point but I favor specifying the `loc` and `scale` for readability even if we're going to use the defaults, especially in a tutorial

Reply:
addressed in https://github.com/CDCgov/multisignal-epi-inference/pull/302/commits/82899592f5f95f7572782c357d7bad4ebca85b25